templates/taxi/car_detail.html

```diff
@@ -12,8 +12,23 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
-  <hr>
+  <div style="display: flex; justify-content: space-between; align-items: center;">
+    <h1>Drivers</h1>
+    <div>
+        {% if request.user in object.drivers.all %}
```

Comment:
The condition `{% if request.user in object.drivers.all %}` should use `car` instead of `object` to maintain consistency and clarity, as `car` is the context variable used throughout the template. Change it to `{% if request.user in car.drivers.all %}`.